init.el

```diff
@@ -20,7 +20,10 @@
     (message (concat "Your version of Emacs (%s) is too old. "
                      "Spacemacs requires Emacs version %d or above.")
              emacs-version spacemacs-emacs-min-version)
-  (load-file (concat user-emacs-directory "core/core-load-paths.el"))
+  (setq spacemacs-start-directory
```

Comment:
A `defvar` is more appropriate here, and eliminates the need for a manual `boundp` check.
```elisp
(defvar spacemacs-start-directory user-emacs-directory)
```